catalog.yml (#7566)
fishtown-analytics/docs.getdbt.com: website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

dbt supports materializing the table in Iceberg table format in two different ways:

- The model configuration field `table_format = 'iceberg'` (legacy)

> - Catalog integration can be configured in the SQL config (inside the `.sql` model file), property file (model folder), or project the file &mdash; [`dbt_project.yml`](/reference/dbt_project.yml)

Reviewer: ```suggestion
- Catalog integration can be configured in the SQL config (inside the `.sql` model file), property file (model folder), or project file ([`dbt_project.yml`](/reference/dbt_project.yml)).
```

:::tip **Catalog integration configuration**

> You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables).

Author: ```suggestion
You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or `dbt_project.yml`.

Refer to our [Snowflake configurations](/reference/resource-configs/snowflake-configs) for more information.
```

> You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables).

Reviewer: Can we expound upon this? It's a little vague and leaves me wondering how it's a tip

dbt supports materializing the table in Iceberg table format in two different ways:

- The model configuration field `table_format = 'iceberg'` (legacy)
- Catalog integration can be configured in the SQL config (inside the `.sql` model file), property file (model folder), or project file ([`dbt_project.yml`](/reference/dbt_project.yml)).

> :::tip **Catalog integration configuration**

Author: ```suggestion
:::info **Catalog integration configuration**
```

> :::info **Catalog integration configuration**

Reviewer: No need for formatting here
```suggestion
:::info Catalog integration configuration
```

You need to create a `catalogs.yml` file to use the integration and apply that integration on the config level.

> Refer to our [Snowflake configurations](/reference/resource-configs/snowflake-configs) for more information.

Reviewer: ```suggestion
Refer to [Snowflake configurations](/reference/resource-configs/snowflake-configs) for more information.
```

> You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or `dbt_project.yml`.

Author: ```suggestion
You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or your `dbt_project.yml`.
```

> You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or your `dbt_project.yml`.

Reviewer 2: > You can also
I'm finding this note a little confusing.  On [this page](https://docs.getdbt.com/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) we say:

> 1. Create a catalogs.yml at the top level of your dbt project.
> 2. Apply the catalog configuration at either the model, folder, or project level.

This excerpt tells me that (at least for managed repos), you **need to** (1) create a catalogs.yml and also (2) apply the configs. But the note indicates it's optional? @amychen1776 you need to define the catalogs.yml, right?

Reviewer 3: That's correct. You need to have a catalogs.yml file to use the integration but then you apply that integration on the config level. 

If this is helpful - the catalogs.yml is similar to a profiles.yml (it tells you where to connect and how like what default warehouse to use) and then into the model/folder/project level, you tell dbt which catalog you want to tie that model to (like config(warehouse = transforming).

> You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or your `dbt_project.yml`.

Reviewer 2: In addition, I think this  "property file, folder, or your `dbt_project.yml`."

Should match what we say above or an abridged version of above? Something like:
> by applying the catalog configuration to the SQL config, property file, or your project file.

> You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or your `dbt_project.yml`.

Author: ```suggestion
You need to create a `catalogs.yml` file to use the integration and apply that integration on the config level.
```

```

> 2. Apply the catalog configuration at either the property file, folder, or your `dbt_project.yml`. <br />

Reviewer 2: @nataliefiann should this match how we say it above?
> SQL config (inside the `.sql` model file), property file (model folder), or project file ([`dbt_project.yml`](/reference/dbt_project.yml))

> 2. Apply the catalog configuration at either the property file, folder, or your `dbt_project.yml`. <br />

Author: ```suggestion
2. Apply the catalog configuration in either the SQL config (inside the .sql model file), property file (model folder), or your `dbt_project.yml`. <br />
```
